docs/content/preview/get-started-guide.md

```diff
@@ -0,0 +1,204 @@
+<!---
+title: How to get started with YugabyteDB
+headerTitle: How to get started with YugabyteDB
+linkTitle: Guide to resources
+description: Find resources for getting started with YugabyteDB.
+headcontent: Find resources for getting started with YugabyteDB
+type: docs
+unversioned: true
+private: true
+--->
+
+Find resources for getting started, migrating existing databases, using your database, and architecting for scalability, reliability, and security.
+
+<ul class="nav yb-pills">
+
+  <li>
+    <a href="https://cloud.yugabyte.com/signup" class="orange">
+      <img src="/icons/yugabyte.svg"></i>
+      Sign Up
+    </a>
+  </li>
+  <li>
+    <a href="https://download.yugabyte.com/" class="orange">
+      <img src="/icons/database.svg"></i>
+      Download Now
+    </a>
+  </li>
+  <li>
+    <a href="https://communityinviter.com/apps/yugabyte-db/register" class="orange">
+      <img src="/icons/slack.svg"></i>
+      Join Us on Slack
+    </a>
+  </li>
+
+</ul>
+
+## Introducing YugabyteDB
+
+**Video**: [Introducing YugabyteDB: The Distributed SQL Database for Mission-Critical Applications](https://www.youtube.com/watch?v=j24p07Frw00)
+: Learn about YugabyteDB and how it supports mission-critical applications.
+
+**Documentation**: [Key benefits](../features/)
+: Why YugabyteDB should power your next cloud-native application.
+
+**Article**: [Distributed SQL 101](https://www.yugabyte.com/distributed-sql/)
+: How Distributed SQL powers database modernization and cloud-native applications.
+
+**Article**: [How to Scale a Single-Server Database: A Guide to Distributed PostgreSQL](https://www.yugabyte.com/postgresql/distributed-postgresql/)
+: Why you need Distributed PostgreSQL to truly scale.
+
+**Article**: [Distributed SQL vs Sharded Postgres](https://www.linkedin.com/pulse/distributed-things-postgresql-franck-pachot-4fr7e/)<br>
```

Comment:
I prefer having a link followed by the description rather than two links and then a common description.